modules/administration-guide/partials/proc_enabling-readiness-init-containers-for-the-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="enabling-readiness-init-containers-for-the-olm-installer_{context}"]
+= Enabling the readiness init containers for the OLM installer
+
+The readiness init containers are not enabled by default, so to use them you first have to enable them. To enable the readiness init containers for {prod-short} installed by the OLM installer:
+
+.Prerequisites
+
+* {prod-short} is installed by the OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
```

Comment:
```suggestion
. Get the {prod-short} Operator subscription name:
```

Reply:
The same as above

Reply:
I understand. In such case, please check if this is also the case in the next step, where I already swapped `find` with `get`.

Reply:
ok